### DISCUSSION TITLE
[Feature Request] DAC Output

### DISCUSSION THREAD

**Discussion by dplischke** [arendst/Tasmota#10427] [Q&A] [answered]
Hi,

I would like to use the ESP32 DAC on GPIO 25 and 26 to generate an analog output. Is there a chance to get it implemented in a (soon) comming version? Or give me a clou where to integrate it as myselfe. But I'm almost a copy and paste programmer ;-)

Thanks a lot, Dominik.

**Comment by Jason2866** [comment]
No because https://github.com/arendst/Tasmota/issues/10036#issuecomment-737078450

**Comment by gemu2015** [comment, marked as answer]
you may use PWM as DAC. it has 10 bit resolution (int DAC has 8)
you only need an R/C to smooth the signal.
scripting supports up to 8 PWM channels on ESP32

**Reply by dplischke** [reply]
Thanks gemu2015. I'll test it.
